### PR TITLE
#17863: Add sweep test for Batch Norm

### LINKED ISSUE #17863: Sweep test for Batch Norm
- Add sweep test for Batch Norm for training and inference mode

### PR DESCRIPTION
**Pull request by VirdhatchaniKN**
### Ticket
#17863

### Problem description
Add sweep test for BN 

### What's changed

- Added sweep test for BN totest for inference mode and training mode

Screenshot 2025-03-06 at 10 06 28 AM

- Added a test_batch_norm method to test locally via pytest

Screenshot 2025-03-07 at 11 44 23 PM

### Checklist
- [x] [ttnn - Run sweeps](https://github.com/tenstorrent/tt-metal/actions/runs/13734818993)

### CODE REVIEW

File: tests/sweep_framework/sweeps/normalization/batch_norm/batch_norm.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        "check_var": [True, False],
+        "weight": [True, False],
+        "bias": [True, False],
+        "eps": [1.0, 0.0, 2.34, 1e-05],
+        "momentum": [0.0, 0.1, 0.5],
+    },
+}
+
+
+def run(
```
Comment (bbradelTT) [inline tests/sweep_framework/sweeps/normalization/batch_norm/batch_norm.py:40]: Please add a test_batch_norm method to test locally via pytest.

See tests/sweep_framework/sweeps/matmul/short/matmul_traces.py
for an example.

**Review by bbradelTT — APPROVED** [review]

**Review by sjameelTT — APPROVED** [review]

**Review by sjameelTT — APPROVED** [review]
Could you run the sweep on pipelines?

**Comment by VirdhatchaniKN** [comment]
https://github.com/tenstorrent/tt-metal/actions/runs/13746596461 - passed
